Improvements on CRUD console command

hbugdoll opened the pull request:
### What are you changing/introducing

- adopted guideline wording to PHPDoc comments (`API`, `CRUD`, `NgRest`, ...)
- fixed typos `camlize`/`Camlized` --> `camelize`/`Camelized` and `Summery` --> `Summary`
- improvements on console output, e.g. clarify the output of view file `build_summary.php`

nadar reviewed (changes requested):
Unit tests are failing, maybe some methods are used in the unit tests. Also i think the "main" output of the console command will be checked, which has changed now, so you need to adjust the tests as well.

hbugdoll commented:
> Unit tests are failing, maybe some methods are used in the unit tests. Also i think the "main" output of the console command will be checked, which has changed now, so you need to adjust the tests as well.

Thanks for this advice. I've adjusted the unit test (and fixed additionally a little type `endpoit` --> `endpoint` there).
